annotation-web/src/main/java/eu/europeana/annotation/web/validation/BaseAnnotationValidator.java

```diff
@@ -524,10 +533,38 @@ void validateEdmRights(Body body, Authentication authentication)
       licence = licence.substring(0, versionStart);
     }
     Set<String> rights = getConfiguration().getAcceptedLicenceses();
-    if (!rights.contains(licence))
-      throw new RequestBodyValidationException(body.getInputString(),
-          I18nConstants.INVALID_PARAM_VALUE, new String[] {BODY_EDM_RIGHTS, rightsClaim});
-
+    if (!rights.contains(licence)) {
```

Comment:
this needs to be updated, the acceptedLicences is the list of open lincences indicated in the ticket. The licences from this list needs to be approved.